data/102/527/513/102527513.geojson

```diff
@@ -182,9 +182,10 @@
     ],
     "oa:elevation_ft":"13",
     "oa:type":"large_airport",
-    "src:geom":"uszcta",
+    "src:geom":"ccsf-sfo",
```

Comment:
Confirmed we already have this source here: https://github.com/whosonfirst/whosonfirst-sources/blob/50557d12f81b871d9c76de826a85d193a07b96cd/sources/ccsf-sfo.json.